MystenLabs/sui
# [Wallet] add SuiSecBlocklist to scan deceptive domain

## Description 

1. [Wallet] add SuiSecBlocklist to scan deceptive domain
2. [Documentation] "struct" for creating nft guide code should use "public struct"
3. Contributed by SuiSec Team

## Test plan 

1. Use this commits, cd to ./apps/wallet ,  run 'pnpm dev' 
2. After building the app, the extension needs to be installed to Chrome from the dist/ directory.
4. Visit: [suiet guardians](https://github.com/suiet/guardians/blob/main/src/domain-list.json) , choose a domain, for example: [zksynks.com](http://zksynks.com/) , open it in a new browser window
5. then you will see: 
![image](https://github.com/MystenLabs/sui/assets/24224985/53030d95-71f8-45a0-aff9-276882f86c48)
6. If user understand the risks and still want to proceed, they can click continue button to close the warning


---

## Release notes

- [ ] Wallet: add SuiSecBlocklist to scan deceptive domain
